src/gg.regression.unity.experimental.runespawn/Editor/Runtime/Runespawn.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "RegressionGames.Runespawn",
+    "rootNamespace": "",
+    "references": [
+        "GUID:c6e66b29de2c44f90985c972c4b1ebc6"
```

Comment:
The Runespawn.asmdef points to RegressionGames.
The Runespawn.Editor points to both Runespawn.asmdef and RegressionGames.

This how the rest of the options look in case I should change something on that level: 
![image](https://github.com/user-attachments/assets/4d3e052b-fee8-4a1b-b861-1a2ff85007dd)

Reply:
Ok, for our other extensions, we use this pattern to be more clear to developers reading the file.

```
    "references": [
        "RegressionGames",
```

Reply:
Switched to using paths over GUID, and added expanded the description.